### ISSUE TITLE
Endianness constants should not changed based on machine endianness

### ISSUE THREAD

**Issue by ErichDonGubler** [nabijaczleweli/registry.pol-rs#1]
I see that in the [`v1` module](https://github.com/nabijaczleweli/registry.pol-rs/blob/596ddab26e35f9b4b77afb8a4195f6e7407bc69a/src/v1/mod.rs#L23) you've named the `REG_DWORD_BIG_ENDIAN` constant equivalent to be `REG_DWORD_ALT`, indicating that it's contrary to the current system's endianness. You even have tests covering this, which is impressive! However, this seems to be a misunderstanding of what these constants mean. When I examine the Windows 7.1 version of `WinNT.h` on my Windows 10 machine, lines 13918-13920 are of particular interest:

```
#define REG_DWORD                   ( 4 )   // 32-bit number
#define REG_DWORD_LITTLE_ENDIAN     ( 4 )   // 32-bit number (same as REG_DWORD)
#define REG_DWORD_BIG_ENDIAN        ( 5 )   // 32-bit number
```

`REG_DWORD_BIG_ENDIAN` ALWAYS corresponds to a big-endian representation, independent of host endianness, and the same applies for `REG_DWORD_LITTLE_ENDIAN`/`REG_DWORD` and little-endian. This means that changing the meaning of different constant values in this library ain't appropriate!

I'm more than happy to make a PR to fix this -- but it should be recognized as an issue first. :)

**Comment by nabijaczleweli** [comment]
Yeah, I wasn't sure what to do with them so I did the most intuitive thing with what I had. My W10 `WinNT.h` says just this (L8147):
```c
#define REG_NONE (0)
#define REG_SZ (1)
#define REG_EXPAND_SZ (2)
#define REG_BINARY (3)
#define REG_DWORD (4)
#define REG_DWORD_LITTLE_ENDIAN (4)
#define REG_DWORD_BIG_ENDIAN (5)
#define REG_LINK (6)
#define REG_MULTI_SZ (7)
#define REG_RESOURCE_LIST (8)
#define REG_FULL_RESOURCE_DESCRIPTOR (9)
#define REG_RESOURCE_REQUIREMENTS_LIST (10)
#define REG_QWORD (11)
#define REG_QWORD_LITTLE_ENDIAN (11)
```

I'm guessing, then, that `REG_QWORD_LITTLE_ENDIAN` is also just an alias to `REG_QWORD`?

**Comment by ErichDonGubler** [comment]
That's absolutely correct. :)

**Comment by nabijaczleweli** [comment]
Released in [`v1.0.0`](https://github.com/nabijaczleweli/registry.pol-rs/releases/tag/v1.0.0).
